src/test/java/games/strategy/engine/data/annotations/ValidateAttachmentsTest.java

```diff
@@ -292,7 +292,7 @@ private static String validateAttachment(final Class<? extends IAttachment> claz
       // validate that there is a field and a getter
       Field field = null;
       try {
-        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, "m_" + propertyName, false);
+        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, propertyName, false);
```

Comment:
The `fail()` call was commented out in 12981f3 (circa 2012).  So this validation has been ignored for quite some time.

As a result of turning off the test failures, the `testAllAttachments()` test is broken, as it currently doesn't find any attachment classes to validate because the compilation output folder changed at some time in the past (probably during the conversion to Gradle).